modules/ztp-validating-the-generation-of-policy-crs.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-validating-the-generation-of-policy-crs_{context}"]
+
+= Validating the generation of policy CRs
+
+Policy custom resources (CRs) are generated in the same namespace as the PolicyGenTemplate from which they were created. The same troubleshooting flow applies to all policy CRs generated from PolicyGenTemplates regardless of whether they are common, group, or site based.
+
+Check the status of the policy CRs:
+
+[source,terminal]
+----
+$ export NS=<namespace>
+----
+
+[source,terminal]
+----
+$ oc get policy -n $NS
+----
+
+The returned output displays the expected set of policy wrapped CRs. If no object is returned, use the following procedure to troubleshoot the ArgoCD pipeline flow from SiteConfig to the policy CRs.
+
+.Procedure
+
+. Check the synchronization of the PolicyGenTemplate to the hub cluster:
++
+[source,terminal]
+----
+$ oc get policygentemplate -A
+----
+or
++
+[source,terminal]
+----
+$ oc get policygentemplate -n $NS
+----
++
+If the PolicyGenTemplate is not synchronized, one of the following situations has occurred:
++
+* The clusters application failed to synchronize the CR from the Git repository to the hub. Use the following command to verify this:
++
+[source,terminal]
+----
+$ oc describe -n openshift-gitops application clusters
+----
++
+Check for `Status: Synced` and that the `Revision:` is the SHA of the commit you pushed to the subscribed repository.
++
+* The pre-sync hook failed, possibly due to failure to pull the container image. Check the ArgoCD dashboard for the status of the pre-sync job in the *clusters* application.
```

Comment:
a failure? Should we have an article here?

Reply:
fixed